app/src/main/java/com/example/travelapp/Dijkstras.java

```diff
@@ -79,18 +88,22 @@ public static Graph createGraph(List<String> ids, Itinerary itinerary) throws In
      * Input graph is guaranteed to be valid and have no negative-weighted edges.
      */
     public static List<Integer> getShortestPath(Graph graph, int src, int tgt) {
-        List<Integer> list = new ArrayList<Integer>();
-        // hard coded base case of size = 2 since no edge between 0 and 1 is added in the graph.
-        if (graph.getSize() == 2) {
-            list.add(0);
-            list.add(1);
+        List<Integer> list = new ArrayList<>();
+        // hard coded base case of size 1
+        if (src == tgt) {
+            list.add(src);
             return list;
         }
-        if (src == tgt) {
+        // hard coded base case of size = 2 since no edge between 0 and 1 is added in the graph.
+        if (graph.getSize() == 2) {
             list.add(src);
+            list.add(tgt);
             return list;
         }
         int size = graph.getSize();
+        // switched distance to a double since with the new graph, normalizing the weights makes it
```

Comment:
nit: instead of saying why you did something say the reason it is that way. for example, instead of "switched distance to a double since..." it would be better to say something like "distance is a double because..."